lib/portalUtils.js

```diff
@@ -50,7 +50,7 @@ _.extend(exports, {
     },
     getPom: function(c, p) {
         p = p || '';
-        var pomPath = path.join(c.config._local.path, p, '/pom.xml');
+        var pomPath = path.join(c.config.bbrc.path, p, '/pom.xml');
```

Comment:
https://github.com/Backbase/bb-cli/blob/master/docs/lib.config.md
/path/to/the/portalserver 
They are not used anywhere, except for my experiments